examples/Azure_IoT_Hub_Arduino_Nano_RP2040_Connect/Azure_IoT_Hub_Arduino_Nano_RP2040_Connect.ino

```diff
@@ -0,0 +1,591 @@
+/*-----------------------------------------------*/
+
+/*
+ * connectToWifi:
+ * The WiFi client connects, using the provided SSID and password.
+ * The WiFi client synchronizes the time on the device. 
+ */
+void connectToWiFi() 
+{
+  logString = "Attempting to connect to WIFI SSID: ";
+  LogInfo(logString + IOT_CONFIG_WIFI_SSID);
+
+  while (WiFi.begin(IOT_CONFIG_WIFI_SSID, IOT_CONFIG_WIFI_PASSWORD) != WL_CONNECTED) 
+  {
+    Serial.print(".");
+    delay(IOT_CONFIG_WIFI_CONNECT_RETRY_MS);
+  }
+  Serial.println();
+
+  logString = "WiFi connected, IP address: ";
+  LogInfo(logString + WiFi.localIP() + ", Strength (dBm): " + WiFi.RSSI());
+  LogInfo("Syncing time.");
+
+  while (getTime() == 0) 
+  {
+    Serial.print(".");
+    delay(500);
+  }
+  Serial.println();
+
+  LogInfo("Time synced!");
+}
+
+/*
+ * initializeAzureIoTHubClient:
+ * The Azure IoT SDK for C client uses the provided hostname, device id, and user agent.
+ */
+void initializeAzureIoTHubClient() 
+{
+  LogInfo("Initializing Azure IoT Hub client.");
+
+  az_span hostname = AZ_SPAN_FROM_STR(IOT_CONFIG_IOTHUB_FQDN);
+  az_span deviceId = AZ_SPAN_FROM_STR(IOT_CONFIG_DEVICE_ID);
+
+  az_iot_hub_client_options options = az_iot_hub_client_options_default();
+  options.user_agent = AZ_SPAN_FROM_STR(IOT_CONFIG_AZURE_SDK_CLIENT_USER_AGENT);
+
+  int result = az_iot_hub_client_init(&azIoTHubClient, hostname, deviceId, &options);
+  if (az_result_failed(result)) 
+  {
+    logString = "Failed to initialize Azure IoT Hub client. Return code: ";
+    LogError(logString + result);
+    exit(result);
+  }
+
+  logString = "Azure IoT Hub hostname: ";
+  LogInfo( logString + IOT_CONFIG_IOTHUB_FQDN);
+  LogInfo("Azure IoT Hub client initialized.");
+}
+
+/*
+ * initializeMQTTClient:
+ * The MQTT client uses the client id and username from the Azure IoT SDK for C client.
+ * The MQTT client uses the generated password (the SAS token).
+ */
+void initializeMQTTClient() 
+{
+  LogInfo("Initializing MQTT client.");
+  
+  int result;
+
+  result = az_iot_hub_client_get_client_id(
+      &azIoTHubClient, mqttClientId, sizeof(mqttClientId), NULL);
+  if (az_result_failed(result)) 
+  {
+    logString = "Failed to get MQTT client ID. Return code: ";
+    LogError(logString + result);
+    exit(result);
+  }
+  
+  result = az_iot_hub_client_get_user_name(
+      &azIoTHubClient, mqttUsername, sizeof(mqttUsername), NULL);
+  if (az_result_failed(result)) 
+  {
+    logString = "Failed to get MQTT username. Return code: ";
+    LogError(logString + result);
+    exit(result);
+  }
+
+  generateMQTTPassword(); // SAS Token
+
+  mqttClient.setId(mqttClientId);
+  mqttClient.setUsernamePassword(mqttUsername, mqttPassword);
+  mqttClient.onMessage(onMessageReceived); // Set callback for C2D messages
+
+  logString = "MQTT Client ID: ";
+  LogInfo(logString + mqttClientId);
+  logString = "MQTT Username: ";
+  LogInfo(logString + mqttUsername);
+  logString = "MQTT Password (SAS Token): ";

```

Comment:
Please just remove the line that prints the password.
Follow as done in this sample: https://github.com/Azure/azure-sdk-for-c-arduino/blob/main/examples/Azure_IoT_Hub_ESP32/Azure_IoT_Hub_ESP32.ino#L233